osu.Game/Overlays/Mods/ModSelectScreen.cs

```diff
@@ -153,6 +156,14 @@ private void load()
             }
         }
 
+        private ColumnDimContainer createModColumnContent(ModType modType, Key[]? toggleKeys = null)
+            => new ColumnDimContainer(CreateModColumn(modType, toggleKeys))
+            {
+                AutoSizeAxes = Axes.X,
+                RelativeSizeAxes = Axes.Y,
+                RequestScroll = column => columnScroll.ScrollTo(Math.Clamp(column.DrawPosition.X - 70, 0, columnScroll.ScrollableExtent))
```

Comment:
Where does the `-70` come from here?

Ignoring that though, scrolling seems weird. Going to the right, it scrolls too far. Also I'm unsure why the clamp should be required. I'd hope to have used `ScrollIntoView` here as that's the behaviour I'd expect (scroll enough to bring the column on to the screen, not to the centre) but that doesn't seem to work as expected.

Rational for this requirement is that if there are more columns, the user should be able to click again in the place they clicked the first column to bring the next into view (since it should generally jump by one column at a time).

Reply:
The -70 thing was done with the intention of bringing the clicked column level with the left edge of the popup at the top of the screen.

Scroll bounds are extended by 70 units on both sides on top of the minimum viable width for the same reason (and also to generally allow the columns some breathing space from the sides rather than force them to be pixel-perfect on the edges of the scfoll).

I did originally attempt to use `ScrollIntoView()` but as you say it doesn't work as expected with shear.

Reply:
I've added a helper method to allow using `ScrollIntoView`. I figure we'll probably want this to be added framework side (it's useful for cases you want to scroll to an item "plus a bit", to show the next thing in the view for instance) but I've added locally for now to avoid dependencies.